eng/pipelines/libraries/stress/http.yml

```diff
@@ -45,8 +45,17 @@ jobs:
   
   - bash: |
       cd '$(httpStressProject)'
+      export HTTPSTRESS_CLIENT_ARGS="-http 2.0"
```

Comment:
How/when is the `-maxExecutionTime` passed to the pipeline? Won't this `export` overwrite any existing value in the arg? Shouldn't it rather be something like:
```
export HTTPSTRESS_CLIENT_ARGS="$HTTPSTRESS_CLIENT_ARGS -http 2.0"
```

Reply:
Overwriting is fixed, but it seems it doesn't pass `-maxExecutionTime` because I don't see a failure summary in the logs.

Reply:
Oh, I was wrong. [It is passed](https://dev.azure.com/dnceng/public/_build/results?buildId=794778&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=e0810b27-de1e-5f7f-0df7-1a1042e925c2&l=1148).

Reply:
Anyways it's now concatenated, so problem should be gone.